plugins/policy/configurator/configurator_impl.go

```diff
@@ -545,6 +561,15 @@ func (cr ContivRules) Copy() ContivRules {
 	return crCopy
 }
 
+// Copy creates a deep copy of PodIPAddresses.
+func (pa PodIPAddresses) Copy() PodIPAddresses {
+	paCopy := make(PodIPAddresses, len(pa))
+	for pod, addr := range pa {
+		paCopy[pod] = addr
```

Comment:
I didn't quite follow why we needs to hold a shallow copy, you might want deep copy if you want no reference back to old object